scenarios/lourys/pterodactyl-wings-bf.yaml

```diff
@@ -0,0 +1,50 @@
+####################
+## Invalid format ##
+####################
+type: leaky
+name: lourys/pterodactyl-wings-bf
+description: "Detect invalid_format ssh bruteforce"
+filter: "evt.Meta.log_type == 'invalid_format'"
+leakspeed: "15s"
+references:
```

Comment:
can you remove the reference ? it was present in ssh to showcase the possiblity

Reply:
Oh, okay!